rabbithole_test.go

```diff
@@ -265,6 +265,20 @@ var _ = Describe("Rabbithole", func() {
 		})
 	})
 
+	Context("PUT /cluster-name", func() {
+		It("Set cluster name", func() {
```

Comment:
Yup. I don't think we depend on cluster name anywhere. `rabbit@localhost` is the value I'd use.